handler/category-dimension-import.go

```diff
@@ -128,46 +128,17 @@ func (h *CategoryDimensionImport) Handle(ctx context.Context, e *event.CategoryD
 		"dimension": e.DimensionID,
 	})
 
-	// Count the total number of dimension options for this instance, and update the instance state to 'completed' if all values have been imported
+	// Count the total number of dimension options for this instance, and update the instance state to 'edition-confirmed' if all values have been imported
```

Comment:
Should this be snake_case, here and everywhere for these sorts of things ?
(just asking)

Reply:
Yes, I agree.
But unfortunately this was defined in dataset API:
https://github.com/ONSdigital/dp-dataset-api/blob/12c3a1ffae2c84d107d89f9010d0454df71fc02e/models/state.go
And it needs to match (also, that is why `dp-api-clients-go` define constants with the correct values)

Reply:
Maybe in a separate task we could update this to snake_case, for dataset-api and everywhere where it's used.